### DISCUSSION TITLE
Json Schema Validation?

### DISCUSSION THREAD

**Discussion by justinmchase** [denoland/deno#11472] [Q&A]
I can't seem to find a 3rd party module or a std library module for deno for simply doing json schema validation. I'm wondering if anyone has done this in Deno yet and if anyone has a recommendation?

**Comment by justinmchase** [comment]
While I couldn't find a straight forward port of ajv or one of the other popular jsonschema validators... I was pleasantly surprised to find that there is a new schema validation standard called JTD (json type definition) and there is a module for this already:

https://deno.land/x/jtd

I actually kind of hate the json schema standard so I'm glad to see this and so long as it works, its a welcome shift!

**Comment by Soremwar** [comment]
Or just use AJV? https://cdn.skypack.dev/ajv@6.12.3?dts

**Reply by justinmchase** [reply]
I actually tried that, I'm kinda new to this so I probably screwed it up but I straight up couldn't figure out how to get it to work. It seems to bork the exports and it wasn't working as the docs suggested and then I just found the other tool which seems to work perfect out of the box so I gave up.
